package.json

```diff
@@ -88,7 +88,7 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "test": "jest",
-    "copy:assets": "cpx 'src/assets/**' 'dist/assets'"
+    "copy:assets": "npx cpx \"src/assets/**\" \"dist/assets\""
```

Comment:
`cpx` is a `devDependency` so I thing the original command should work. Why did you change it? What issue are you going to fix?

Reply:
Unfortunately, the original command doesn nothing. If you try the original one running "npm run copy:assets" you'll see that the folder "images" is never copied inside the assets folders from "dist" (and this caused us a lot of issues on one of the components on B2BP). After several tries, this is the one that worked 100% of the times and we need the "images" folder for this PR due to the badges we implemented.

I'm not a fan to change scrips, but seemed that no one tested that one and thinked that worked only 'cause 2 folders on 3 of them were copied

Reply:
Checking the [last version](https://www.npmjs.com/package/@pagopa/pagopa-editorial-components/v/3.1.2?activeTab=code) I see the folder `images` and its content seems right to me:

![Screenshot 2024-03-26 at 15 04 37](https://github.com/pagopa/pagopa-editorial-components/assets/4775679/fb310dfd-c629-4aa7-a4e0-2778382583d0)